internal/pipeline/cmd.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: 2019

There should be some tooling to catch these :)

Reply:
I believe this line is added by some tool :(. It's added after running `make all`.

Reply:
Added by this line

https://github.com/GoogleContainerTools/kpt/blob/739cb5e352b599a96f6f3e33e7593489c6481fd9/scripts/update-license.sh#L18